README.md

```diff
@@ -57,6 +57,10 @@ Then, compile the code into an executable file and install it (installs in /usr/
 sudo make install
 ```
 
+Conversely, if you ever (lose your mind and) want to uninstall, you can run:

```

Comment:
Thanks for this! Perhaps, despite being right 😆, it'd be better to remove `(lose your mind and)`.
Do amend your commit to keep a clean history.

T